### PR TITLE
Add libra_chain_id for all JSONRPC Response

### PR DESCRIPTION
**Pull request by thefallentree**
This will allow client to detect it has connect to the right fullnode.

### CODE REVIEW

File: json-rpc/src/runtime.rs [hunk 1]
```diff
@@ -93,6 +102,7 @@ async fn rpc_endpoint(
     data: Value,
     service: JsonRpcService,
     registry: Arc<RpcRegistry>,
+    chain_id: ChainId,
```
Comment (christinacelee) [inline json-rpc/src/runtime.rs:105]: can we store chain_id in JsonRpcService instead?

Comment (thefallentree) [reply json-rpc/src/runtime.rs]: yeah, good idea, fixed.

File: json-rpc/src/tests/unit_tests.rs [hunk 2]
```diff
@@ -189,6 +189,7 @@ fn test_json_rpc_protocol() {
     let data: JsonMap = resp.json().unwrap();
     assert!(data.get(JSONRPC_LIBRA_LEDGER_VERSION).is_some());
     assert!(data.get(JSONRPC_LIBRA_LEDGER_TIMESTAMPUSECS).is_some());
+    assert!(data.get(JSONRPC_LIBRA_CHAIN_ID).is_some());
```
Comment (christinacelee) [inline json-rpc/src/tests/unit_tests.rs:192]: can we go further and check that chain_id == ChainId::test?

Comment (thefallentree) [reply json-rpc/src/tests/unit_tests.rs:192]: done.

**Pull request closed without merging**